src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Excavator extends Machine{
+
+    @Override
+    public void doWork() {
+        System.out.println("Excavator started its work ⚙.");

```

Comment:
The message printed should be more informative about the type of machine. Instead of a generic message, specify that this is the Excavator that has started working, in accordance with the checklist requirement to write informative messages in methods.